checks.d/consul.py

```diff
@@ -0,0 +1,215 @@
+# 3p
+import requests
+
+# stdlib
+import time
+
+# project
+from checks import AgentCheck
+
+
+class ConsulCheck(AgentCheck):
+    CONSUL_CHECK = 'consul.up'
+    HEALTH_CHECK = 'consul.check'
+
+    CONSUL_CATALOG_CHECK = 'consul.catalog'
+    CONSUL_NODE_CHECK = 'consul.node'
+
+    def __init__(self, name, init_config, agentConfig, instances=None):
+        AgentCheck.__init__(self, name, init_config, agentConfig, instances)
+
+        self._last_known_leader = None
```

Comment:
This is not multi instance safe, but i guess, i don't see a case where one would be running multiple consul clusters. 
Can you check that similarly to what is done in the disk check ? https://github.com/DataDog/dd-agent/blob/master/checks.d/disk.py#L26-L27